### DISCUSSION TITLE
Non-sensible value for sm:contentLanguage in most FIBO ontologies

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1382] [General]
Pretty much all FIBO ontologies declare a sm:contentLanguage of 	http://www.w3.org/standards/techs/owl#w3c_all 
However that now redirects to https://www.w3.org/TR/?tag=data#w3c_all which gives a page listing *all* 150 or so data-related W3 specifications which I'm sure was not the intent.
The specification for the property sm:contentLanguage says "a link to the core language and/or specification of the model/electronic file, for example, UML, OWL, SysML, BPMN, including linking to the version of the language in question"
In which case surely the URL should be http://www.w3.org/2002/07/owl#

**Comment by VladimirAlexiev** [comment]
There's also https://www.w3.org/ns/formats/, does it make any sense to mark different release formats with some of these? Perhaps no because it's in the filename.

**Comment by rivettp** [comment]
@VladimirAlexiev those represent specific serializations, and the idea here is to represent OWL in general. @ElisaKendall has started to use a link to the OWL Quick Reference https://www.w3.org/TR/owl2-quick-reference/ but I still prefer the OWL namespace URI. Any thoughts?

**Comment by VladimirAlexiev** [comment]
Agree, the namespace is better. And if possible, use a real URL, not a literal with datatype AnyURI
